presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileBasedSecurity.java

```diff
@@ -36,27 +37,28 @@ public void setUp()
             throws Exception
     {
         String path = this.getClass().getResource("security.json").getPath();
-        queryRunner = createQueryRunner(getTables(), ImmutableMap.of(), "file", ImmutableMap.of("security.config-file", path));
+        queryRunner = createQueryRunner(ImmutableList.of(NATION), ImmutableMap.of(), "file", ImmutableMap.of("security.config-file", path));
     }
 
     @AfterClass(alwaysRun = true)
     public void tearDown()
     {
         queryRunner.close();
+        queryRunner = null;
```

Comment:
Why this null? Is reference to test class kept after tests are run?

Reply:
yes, it is

Reply:
i made changes in other classes as well -- https://github.com/prestodb/presto/pull/10866